DemoServer/Controllers/Demos/Attachments/IndexAttachmentDetails/metadata.json

```diff
@@ -0,0 +1,160 @@
+{
+    "Slug": "index-attachment-details",
+    "SourceFileName": "IndexAttachmentDetailsController.cs",
+    "Title": "Index Attachment Details",
+    "DescriptionLines": [
+        "**attachment's details and content can both be indexed**.",
+        "",
+        "This demo shows indexing of attachment's details."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Store Attachment Demo",
+            "Type" : "demo",
+            "Url"  : "/attachments/store-attachment"
+        },
+        {
+            "Title": "Indexing Attachments",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/indexing"
+        },
+        {
+            "Title": "What are attachments",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/document-extensions/attachments/what-are-attachments"
+        },
+        {
+            "Title": "Attachments and binary data",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/3-document-modeling#attachments-and-binary-data"
+        },
+        {
+            "Title": "Working with attachments",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/4-deep-dive-into-the-ravendb-client-api#working-with-attachments"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define the index class",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* This example defines a Map index on the _Employees_ collection."
+            ],
+            "Assets": [
```

Comment:
![image](https://user-images.githubusercontent.com/507133/148106876-ab927b52-b56b-46a8-a0cb-242b5e793d0a.png)

Do you think that small padding below the line would improve looks?

Reply:
There is this issue I opened 2 weeks ago about that
https://issues.hibernatingrhinos.com/issue/HRINT-2468